cob_default_env_config/ipa-apartment/polygon_maps/restriction_polygon.yaml

```diff
@@ -0,0 +1,4 @@
+restricted areas: 
+  - id: 31287680
```

Comment:
what does this parameter actually do?
does it 
 1. restrict the robot to only move __within__ the specified polygon or 
 1. restrict the robot from moving __into__ the specified polygon